src/text-editor-component.coffee

```diff
@@ -319,26 +321,27 @@ class TextEditorComponent
     if @mounted
       @presenter.setFocused(false)
 
+  onKeyPress: (event) =>
+    @detectedKeyPress = true
+
   onTextInput: (event) =>
     event.stopPropagation()
-
-    # If we prevent the insertion of a space character, then the browser
-    # interprets the spacebar keypress as a page-down command.
-    event.preventDefault() unless event.data is ' '
+    event.preventDefault()
```

Comment:
@hellendag Removing this guard caused a regression where we interpret spaces as page downs (per the comment). After restoring the code preventing the `preventDefault` call for spaces, IME and the press-and-hold menu still seem to work. Am I missing anything?